templates/zerver/help/add-a-bot-or-integration.md

```diff
@@ -0,0 +1,39 @@
+# Add a Bot or Integration
+
+Bots and integrations are features that are accesible to all members of an organization.
+
+## To Add a Bot
+
+{!go-to-the.md!} [Your Bots](/#settings/your-bots)
+{!settings.md!}
+
+2. On this page, under the **Add a New Bot** view, you can enter a name, an email, and
+optionally an avatar for your bot. Then click the **Create Bot** button to create your bot.
+
+    ![Bots page](/static/images/help/add_a_new_bot.png)
+
+3. An entry such as the following should appear above the **Add a New Bot** view, confirming
+the creation of your bot.
+
+    ![Example bot](/static/images/help/bot_example.png)
+
+4. You now have access to your bot's API key and API
+configuration file (`.zuliprc`) which you will need for integrations that you would like
+to use with this bot. You can click on the download icon
+(![Download icon](/static/images/help/download_button.png)) to download your `.zuliprc` file.
+
+## To Add an Integration
+
+There is a full list of integrations and the directions to add them on the Integrations page.
+You can access this page by following the given instructions.
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
```

Comment:
We removed that image; check the style guide for how to grab icons properly now :)